set.mm

```diff
@@ -30769,8 +30769,17 @@ Such interpretation is rarely needed (see also ~ df-ral ).  (Contributed
   df-v $a |- _V = { x | x = x } $.
 
   $( All setvar variables are sets (see ~ isset ).  Theorem 6.8 of [Quine]
-     p. 43.  (Contributed by NM, 26-May-1993.) $)
+     p. 43.  (Contributed by NM, 26-May-1993.)  Remove use of ~ ax-12 .
+     (Revised by SN, 28-Aug-2023.) $)
   vex $p |- x e. _V $=
+    ( cv weq cab cvv wcel wsb eqid sbt df-clab mpbir df-v eleqtrri ) ABZAACZADZ
+    ENPFOAAGOAANHIOAAJKALM $.
```

Comment:
Use equid instead of eqid. Also, I will move to Main bj-vexw/sn-vexw and I'll make vex use it. I'll do it once this is merged